common/app/layout/Breakpoint.scala

```diff
@@ -0,0 +1,19 @@
+package layout
+
+import scalaz.syntax.std.option._
+
+sealed trait Breakpoint {
+  val minWidth: Option[Int]
+}
+
+case object Mobile extends Breakpoint {
+  val minWidth = None
+}
+
+case object Tablet extends Breakpoint {
+  val minWidth = 740.some
```

Comment:
Yeah, I think in the cases where it's needed, use it, otherwise the simplest idiomatic line would be more reader-friendly. Of course what's reader-friendly evolves with the codebase...